themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -138,7 +132,7 @@ JS;
       <div class="modal-dialog">
         <div class="modal-content">
           <button type="button" class="close" data-dismiss="modal">
-            <svg aria-label="<?=$this->transEscAttr('Close') ?>" viewBox="0 0 24 24"><path d="M6 18L18 6M6 6l12 12"></path></svg>
+            <svg aria-label="<?=$this->transEscAttr('Close')?>" viewBox="0 0 24 24"><path d="M6 18L18 6M6 6l12 12"></path></svg>
```

Comment:
I don't necessarily disagree with these white-space changes, but I am curious if they came from a style guide or a personal preference. As the progressive theme is built, I want to make sure the formatting is most agreeable.

Reply:
It would be nice to see if we could find a php-cs-fixer rule to apply consistent formatting, if we have a preference one way or the other.

Reply:
I'm not seeing anything in the obvious places about this. The examples tend to have a space but there's inconsistency with semicolons: `); ?>` vs. `) ?>`.

Nothing in php-cs-fixer, as far as I can tell.

Reply:
One final reference: Laminas documentation seems to use no-semicolon, space at both ends: `<?= $this->htmlTag(['lang' => 'en'])->openTag() ?>`.

Not a fan of the space at the front, but this puts the majority of large projects in the space-please category. https://docs.laminas.dev/laminas-view/helpers/html-tag/.

Sorry to make this about the color of the bikeshed.

Reply:
Yeah, I'm also not a fan of the leading space, but I could live with it if we had tools to apply it -- consistency would be nice. However, it seems you are correct that php-cs-fixer is not going to help us at present; see: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5138

Reply:
@crhallberg They were actually out of the habit. Can tune them back into what they were.

Reply:
@LuomaJuha, that's probably worth doing -- reducing whitespace-based churn will make the commit history easier to read. I'm definitely open to revisiting how we format our short-tags in templates, but I think that's best done globally with an automated tool rather than piecemeal by hand. :-)

Reply:
Yea, I've restored the whitespaces into their original places. : )